### PR TITLE
Docker lesson w2016

### PR DESCRIPTION
**Pull request by ElijahCaine**
@borcean @lucywyman thoughts?

### CODE REVIEW

**Comment by borcean** [comment]
:+1: I still need to look through the exercise in Bootcamp-Exercises#15 but the slides look good!

File: source/bootcamp_content/docker-docker-docker.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+What is Docker?
+---------------
+
+.. figure:: /static/docker_logo.png
+    :align: center
+    :width: 50%
+    :target: https://www.docker.com
+
+A very popular tool for working with containers-like things.
```
Comment (lucywyman) [inline source/bootcamp_content/docker-docker-docker.rst]: container

Ignore if this is only comment

File: source/bootcamp_content/docker-docker-docker.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+.. figure:: /static/vm_vs_container.png
+    :align: center
+    :width: 70%
+    :target: https://www.docker.com
+
+
+What is Docker?
+---------------
+
+.. figure:: /static/docker_logo.png
```
Comment (lucywyman) [inline source/bootcamp_content/docker-docker-docker.rst]: Maybe no image -- doesn't seem necessary and is squished?

File: source/bootcamp_content/docker-docker-docker.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    / # echo "Pretty cool!"
+    Pretty cool!
+    / # exit
+
+
+Docker is not Perfect
+---------------------
+
+There's a lot of security issues related to signing containers and trust. i.e.:
+There is no way to verify a Docker Container is trustworty.
```
Comment (lucywyman) [inline source/bootcamp_content/docker-docker-docker.rst]: trustworthy

File: source/bootcamp_content/docker-docker-docker.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+.. _docker documentation: https://docs.docker.com/
+.. _rkt documentation: https://coreos.com/rkt/docs/latest/
+
+
+Activity
+--------
+
+This week's demo can be found here:
+
+https://github.com/DevOpsBootcamp/Bootcamp-Exercises/tree/master/2015-2016/docker
```
Comment (lucywyman) [inline source/bootcamp_content/docker-docker-docker.rst]: Let's make sure this works before merging

**Pull request merged**